e2e/testcafe-devextreme/tests/accessibility/contextMenu.ts

```diff
@@ -0,0 +1,46 @@
+import { Item, Properties } from 'devextreme/ui/context_menu.d';
+import ContextMenu from 'devextreme-testcafe-models/contextMenu';
+import url from '../../helpers/getPageUrl';
+import { defaultSelector, testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const items: Item[] = [
+  { text: 'remove', icon: 'remove', items: [{ text: 'item_1' }, { text: 'item_2' }] },
+  { text: 'user', icon: 'user' },
+  { text: 'coffee', icon: 'coffee' },
+];
+
+const options: Options<Properties> = {
+  target: [defaultSelector],
+  position: [{ offset: '10 10' }],
+  items: [items],
+  disabled: [true, false],
+  hint: [undefined, 'hint'],
+  selectionMode: ['single', 'none'],
+  submenuDirection: ['auto', 'left', 'right'],
```

Comment:
This option does not affect attributes. We don't care where the submenu is displayed, because the visual component will be the same in any position. We can skip it.